src/main/java/com/amazon/opendistroforelasticsearch/security/configuration/ConfigurationRepository.java

```diff
@@ -124,7 +125,7 @@ public void run() {
 
                         try {
                             String lookupDir = System.getProperty("security.default_init.dir");
-                            final String cd = lookupDir != null? (lookupDir+"/") : new Environment(settings, configPath).pluginsFile().toAbsolutePath().toString()+"/opendistro_security/securityconfig/";
+                            final String cd = lookupDir != null? (lookupDir+"/") : new Environment(settings, configPath).pluginsFile().toAbsolutePath().toString()+"/" + OpenDistroSecurityPlugin.PLUGIN_NAME + "/" + "securityconfig/";
```

Comment:
Did you test this ? Is this working as expected?

Reply:
Tested this and confirmed with Palash. The security plugin configurations have been successfully loaded.